kratos/spatial_containers/tree.h

```diff
@@ -314,61 +335,58 @@ class Tree
         PointerType Result = *mPointsBegin; // NULL ??
         CoordinateType rResultDistance = static_cast<CoordinateType>(DBL_MAX); // DistanceFunction()(ThisPoint,**mPointsBegin);
 
-        // searching the tree
-        mRoot->SearchNearestPoint(ThisPoint,Result,rResultDistance);
+        // Searching the tree
+        mRoot->SearchNearestPoint(ThisPoint, Result, rResultDistance);
 
         return Result;
     }
-    
-    void SearchNearestPoint( PointerType const& ThisPoints, SizeType const& NumberOfPoints, IteratorType &Results, std::vector<CoordinateType> ResultsDistances)
```

Comment:
Why are you removing this function? Is this being covered in other call?

Reply:
That function is not called anywhere and doesn't even compile (I tried when implementing the test)

Reply:
Ok!